examples/Extension/AcmeExtension.php

```diff
@@ -83,7 +83,7 @@ public function load(Container $container): void
         // endsection: executor_di
 
         // section: env_provider_di
-        $container->register(AcmeExecutor::class, function (Container $container) {
+        $container->register(HomeProvider::class, function (Container $container) {
```

Comment:
not really sure how this passed CI but this was using the wrong service name cc @keulinho